Gopkg.lock

```diff
@@ -116,17 +116,6 @@
   revision = "1dc8cf203d24cd454e71ce40ab4cd0bf3112df90"
   version = "v1.27.0"
 
-[[projects]]
-  branch = "master"
-  digest = "1:a74730e052a45a3fab1d310fdef2ec17ae3d6af16228421e238320846f2aaec8"
-  name = "github.com/alecthomas/template"
```

Comment:
I'll clean this up and you can rebase.  I think I'll add a `dep check` step to the CI build too.

Reply:
Did this in 5108e82a